stable/kommander/charts/kubecost/templates/ingress-roles.yaml

```diff
@@ -0,0 +1,52 @@
+{{- if .Values.portalRBAC.enabled }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: opsportal-kommander-kubecost-admin
```

Comment:
:

Reply:
can you repeat the question 😄

Reply:
Please let's add release name to the name of a resource when possible. This will handle the multiple release cases and avoid collisions.

Reply:
lol i thought i wrote 👍 ... but it was a `:` sorry too hasty